packages/pods-core/src/builtin/OrbitPod.ts

```diff
@@ -0,0 +1,341 @@
+        });
+        next = response.data.links.next;
+      });
+    } catch (error: any) {
+      throw new DendronError({
+        message: stringifyError(error),
+        severity: ERROR_SEVERITY.MINOR,
+      });
+    }
+    return { members, next };
+  };
+
+  /**
+   * method to parse members as notes.
+   * - creates new noteprops if note is not already there in the vault
+   * - writes in a temporary hierarchy if the note is conflicted
+   * - updates previously imported notes if there are no conflicts
+   */
+  async membersToNotes(opts: {
+    members: any;
+    vault: DVault;
+    engine: DEngineClient;
+    wsRoot: string;
+    config: ImportPodConfig;
+  }) {
+    const { vault, members, engine, wsRoot, config } = opts;
+    const conflicts: Conflict[] = [];
+    const create: NoteProps[] = [];
+    members.forEach((member: any) => {
+      const { orbitId, name, github, discord, linkedin, twitter, hn, website } =
+        member;
+      member = {
+        custom: {
+          ...config.frontmatter,
+          orbitId,
+          social: {
+            linkedin,
+            github,
+            twitter,
+            discord,
+            hn,
+            website,
+          },
+        },
+      };
+      const noteName = name || github || discord;
+      let fname;
+
+      const note = NoteUtils.getNoteByFnameV5({
+        fname: `people.${noteName}`,
+        notes: engine.notes,
+        vault,
+        wsRoot,
+      });
+
+      if (!_.isUndefined(note)) {
+        if (!_.isUndefined(note.custom.orbitId)) {
+          const conflictData = this.getConflictedData({ note, member });
+          if (conflictData.length > 0) {
+            fname = `people.orbit.duplicate.${Time.now().toFormat(
+              "y.MM.dd"
+            )}.${noteName}`;
+            conflicts.push({
+              conflictNote: note,
+              conflictEntry: NoteUtils.create({ ...member, fname, vault }),
+              conflictData,
+            });
+          } else {
+            this.getUpdatedData({ note, member, engine });
+          }
+        }
+      } else {
+        fname = `people.${noteName}`;
+        create.push(NoteUtils.create({ ...member, fname, vault }));
+      }
+    });
+
+    return { create, conflicts };
+  }
+
+  /**
+   * returns all the conflicted entries in custom.social FM field of note
+   */
+  getConflictedData = (opts: { note: NoteProps; member: any }) => {
+    const { note, member } = opts;
+    const customKeys = Object.keys(note.custom?.social);
+    return customKeys.filter((key: string) => {
+      return (
+        note.custom.social[key] !== null &&
+        member.custom.social[key] !== note.custom.social[key]
+      );
+    });
+  };
+
+  /**
+   * updates the social fields of a note's FM
+   */
+  getUpdatedData = (opts: {
+    note: NoteProps;
+    member: any;
+    engine: DEngineClient;
+  }) => {
+    const { note, member, engine } = opts;
+    const customKeys = Object.keys(note.custom?.social);
```

Comment:
instead of looping through keys of note, we should loop through orbit keys. a note might not have all the social keys that orbit does